openwisp_radius/settings.py

```diff
@@ -6,7 +6,10 @@
 from django.conf import settings
 from django.core.exceptions import ImproperlyConfigured
 
-logger = logging.getLogger(__name__)
+# The logger 'pre_django_setup' is suppose to be
```

Comment:
should be supposed?

Reply:
Didn't know it's supposed to be `supposed` when `suppose` is used phrase in a case like this. Changed it.